docs/modules/sort/overview.md

```diff
@@ -9,24 +9,34 @@ Inlong-sort is simply an Flink application, and relys on Inlong-manager to manag
 
 # features
 
+## supported transforms
+- String split index
+- String regular replace 
+- String regular replace the first value
+- Data filter
+- Data distinct
+- Regular join
+
 ## supported sources
-- pulsar
+- Pulsar
+- MySQL
+- Kafka
 
 ## supported storages
-- hive（Currently we support parquet, orc and text file format）
-- kafka
-- clickhouse
-- iceberg
+- Hive
+- Kafka
+- ClickHouse
+- Iceberg
 
 ## limitations
```

Comment:
do we have this limitation for the lightweight sort now?

Reply:
We do not have this limitation, i remove it latter